tensorflow-jupyter/tensorflow-deployment.yaml

```diff
@@ -78,16 +78,16 @@ spec:
           # If systems with i5 / i9 / Xeon CPUs and/or 1G ethernet are not available to fulfill the requested resources, the Pods
           # will be scheduled on higher end systems.
           preferredDuringSchedulingIgnoredDuringExecution:
```

Comment:
Noting that this section and its explanatory comment above will need to be updated later to match the CPU types we currently have available. Right now it's just commented out.